server/src/event/writer/mem_writer.rs

```diff
@@ -25,20 +25,22 @@ use itertools::Itertools;
 
 use crate::utils::arrow::adapt_batch;
 
+static BUF_SIZE: &str = "P_BUFFER_SIZE";
```

Comment:
@Eshanatnight did you put a default value of 16mb in case P_BUFFER_SIZE is not set

Reply:
Yes, that variable is just there so that Deepsource errors are not there